Fraction.py

```diff
@@ -60,22 +60,16 @@ def denominator(self, new_denominator):
         self._numerator //= gcd
 
     def get_numerator(self):
+        return str(self._numerator)
 
-        gcd = self.gcd(self.numerator, self.denominator)
-        lowest_numerator = self.numerator // gcd
-        return str(lowest_numerator)
 
     def get_denominator(self):
-        gcd = self.gcd(self.numerator, self.denominator)
-        lowest_denominator = self.denominator // gcd
-        return str(lowest_denominator)
+        return str(self._denominator)
+
 
     def get_fraction(self):
         numerator = int(self.get_numerator())
```

Comment:
No need to call `int()` on `self.get_numerator()` and `self.get_denominator()`. Simply use `self.numerator` and `self.denominator` as is instead.